README.Rmd

```diff
@@ -30,7 +30,7 @@ An [R](http://r-project.org) package to support development of R-based projects
 
 ### Features
 
-Currently there are two main functions for auto-populating a new R-based data analysis or package project directory:
+Currently there are two main functions for auto-populating a new R-based data analysis or package project directory with folders & files that encourage best practice in scientific computing and ensure the project has all the [required bcgov items](https://github.com/bcgov/BC-Policy-Framework-For-GitHub/blob/master/BC-Gov-Org-HowTo/Cheatsheet.md):
```

Comment:
"with folders & files that encourage best practice in scientific computing and ensure the project has all the [required bcgov items]" is replicated down in line 68.